pxd.h

```diff
@@ -213,7 +266,15 @@ struct rdwr_in {
 	rdwr_in() = default;
 #endif
 	struct fuse_in_header in;	/**< fuse header */
-	struct pxd_rdwr_in rdwr;	/**< read/write request */
+	union {
+		struct pxd_rdwr_in rdwr;	/**< read/write request */
+		struct pxd_completion completion; /**< user request completion */
+	};
+};
+
+struct rdwr_in_v1 {
+	struct fuse_in_header_v1 in;	/**< fuse header */
+	struct pxd_rdwr_in_v1 rdwr;	/**< read/write request */
```

Comment:
Can this be:
struct rdwr_in_v1 {
	struct fuse_in_header in;	/**< fuse header */
	struct pxd_rdwr_in rdwr;	/**< read/write request */
}
and remove the other v1 specific changes made. They look the same to me, correct me if I am wrong.

Reply:
ok